packages/DatePicker/src/DatePicker.js

```diff
@@ -1,30 +1,24 @@
 import React from "react";
 import PropTypes from "prop-types";
-import moment from "moment";
 import momentPropTypes from "react-moment-proptypes";
 
-import CalendarIcon from "@paprika/icon/lib/Calendar";
-import Input from "@paprika/input";
 import Popover from "@paprika/popover";
-import useI18n from "@paprika/l10n/lib/useI18n";
+import DateInput from "@paprika/date-input";
 import useDebounce from "@paprika/helpers/lib/hooks/useDebounce";
-import usePrevious from "@paprika/helpers/lib/hooks/usePrevious";
 import isElementContainsFocus from "@paprika/helpers/lib/dom/isElementContainsFocus";
 
 import Calendar from "./components/Calendar";
-import DateInput from "./components/DateInput";
+import DateInputPropsExtractor from "./components/DateInput";
```

Comment:
Have we been using this `PropsExtractor` naming convention elsewhere? I agree it's purpose is more clear than if it were `DatePickerInput`, and I'm not usually a fan of something as generic as `Container`, but maybe `PropsContainer` is a better description? `PropVessel` if you prefer something shorter/weirder 😁?

Reply:
Original idea of renaming this component: because we import actual DateInput here, we can't keep the same name for this component.

I just renamed it to `DateInputPropsContainer`. Thanks, @mikrotron.

As of I am not native speaker it is hard for to choose correct term. Another options that I can suggest is `DateInputPropsCollector`. WDYT about it?

Reply:
I think either name will work fine – I prefer both to `Extractor`. 

I actually like `Collector` better personally, just because `Container` gets used for a lot of things, and so is almost meaningless.  But since this is internal to the component, it's not a big deal. You can choose what you like the best.